### ISSUE TITLE
Mod Conflict - More Gems

### ISSUE THREAD

**Issue by Zarepheth** [MehVahdJukaar/SnowySpirit#35]
The latest release of Snowy Spirit, `snowyspirit-1.19.2-2.0.13-fabric.jar`, causes the Moissanite Paxels from [More Gems](https://www.curseforge.com/minecraft/mc-mods/more-gems-fabric) to no longer register as appropriate tools for things that otherwise accept the paxel.

Specifically, targeting blocks that should accept a pickaxe now treats the paxel as a non-pickaxe tool. That means I lost some hard to find resources! Similar issues happened with a block requiring an axe. I did not test far enough to know about hoes or shovels - though I expect similar problems there.

**Comment by MehVahdJukaar** [comment]
I have no idea on how this could affect this. Do you have broken tags in your logs?

**Comment by Zarepheth** [comment]
I also don't know.

However, I just checked my single player world, using the latest version, 
`snowyspirit-1.19.2-2.0.14-fabric.jar`. The Jade indicates that the paxels are appropriate tools for logs (axe), cobblestone (pickaxe), dirt (shovel), and hay bales (hoe).

Between my initial report and now, I have updated various mods, but neither More Gems or Snowy Spirit. It's possible something else was broken and subsequently fixed, fixing this problem.
